libs/ui/src/theme/theme.ts

```diff
@@ -73,6 +73,19 @@ export const defaultDarkTheme: Theme = {
       disabledBg: grayDark.gray9,
     },
   },
+  radio: {
+    item: {
+      bg: indigoDark.indigo3,
+      activeBg: amberDark.amber9,
+      disabledBg: 'rgba(255, 255, 255, 0.034)',
```

Comment:
We'll need to update these to use the correct disabled colors. 
We will address this in[#143](https://github.com/HausDAO/daohaus-monorepo/issues/143)

Reply:
I will leave it as it for now cause I can't find matching colors from the lib

Reply:
Definitely something to bring up to designers.